### PR TITLE
RELATED: RAIL-3566 invalid KPI import

### PR DESCRIPTION
**Pull request by plavemar**
- replace SDK 7 react-component import with SDK-8 sdk-ui one

JIRA: RAIL-3566

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/780/ : SUCCESS in 1m 12s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/498/ : SUCCESS in 7s

File: docs/10_vis__kpi_component.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ A **KPI** \(Key Performance Indicator\) renders a measure calculated by the Good
 ## Structure
 
 ```jsx
-import { Kpi } from "@gooddata/react-components";
+import { Kpi } from "@gooddata/sdk-ui";
 
 <Kpi
     measure="<measure-identifier>"
```
Comment (no23reason) [inline docs/10_vis__kpi_component.md:16]: That is not the whole story, the new component does not support the `format` prop (I cannot put the comment on that line). I think it is safe to remove the line that mentions it.

Comment (no23reason) [inline docs/10_vis__kpi_component.md:16]: Also, `measure` now takes `IMeasure` object, so the whole example should be st like

```jsx
import { Kpi } from "@gooddata/sdk-ui";
import { newMeasure } from "@gooddata/sdk-model";

const measure = newMeasure("<measure-identifier>");

<Kpi
    measure={measure}
    filters={<filters>}
    separators={<separators>}
/>
```

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/781/ : SUCCESS in 1m 00s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/499/ : SUCCESS in 6s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/782/ : SUCCESS in 2m 39s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/500/ : SUCCESS in 20s

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-gate-PR437

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/784/ : SUCCESS in 1m 58s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-fossa-scanning-zuul-el7/138/ : SUCCESS in 37s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sonarqube-scan-zuul-gate/38/ : SUCCESS in 40s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/228/ : SUCCESS in 17s

**Pull request merged**